composer.json

```diff
@@ -9,7 +9,10 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "scripts": {
-        "serve": "./yii serve",
+        "serve": [
+            "Composer\\Config::disableProcessTimeout",
```

Comment:
Maybe run this command into `ServeCommand`, becuase if run `./yii serve` then problem with timeout will be keep.

Reply:
Are you sure? `./yii serve` does not touch `composer`